pkg/util/consts.go

```diff
@@ -1,5 +1,7 @@
 package util
 
 const (
-	OperatorNamespaceName = "openshift-svcat-controller-manager-operator"
+	KubeAPIServerNamespace = "openshift-kube-apiserver"
+	OperatorNamespace      = "openshift-svcat-controller-manager-operator"
+	TargetNamespace        = "kube-service-catalog-controller-manager"
```

Comment:
i'm a little confused between this PR and your other one renaming this, so just make sure you've coordinated these changes so everything stays in sync and you don't end up re-introducing the old value here...

Reply:
Yeah, agreed.  The other change to openshift-service-catalog-controller-manager was late breaking (yesterday) and caused by the need to be in a system level namespace because of the priority class name.

Reply:
rebased and fixed up names spaces.